ansible/roles/root_account/tasks/main.yml

```diff
@@ -84,8 +84,8 @@
     exclusive: '{{ root_account__authorized_keys_exclusive|bool }}'
     state: 'present'
     user: 'root'
-  when: root_account__enabled|bool and root_account__combined_authorized_keys|d() and
-        root_account__authorized_keys_state != 'absent'
+    follow: '{{ root_account__authorized_keys_follow | bool }}'
+  when: root_account__enabled|bool and root_account__authorized_keys_state != 'absent'
```

Comment:
Why remove `root_account__combined_authorized_keys|d()` from the `when:` clause here?